src/entity/DisallowHandle.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, EntityManager, Index, PrimaryColumn } from 'typeorm';
+
+@Entity()
+export class DisallowHandle {
+  @PrimaryColumn({ unique: true })
```

Comment:
A single primary column is unique by default. But I don't mind being explicit about it if you wish